tests/udp/test_raw_traffic_with_fixed_udp_ports.py

```diff
@@ -0,0 +1,78 @@
+from abstract_open_traffic_generator.flow import *
+import pytest
+
+from abstract_open_traffic_generator import flow
+import utils
+
+
+@pytest.mark.parametrize('packets', [1000])
+@pytest.mark.parametrize('size', [74, 128, 1518])
+def test_raw_traffic_with_fixed_udp_ports(api, b2b_raw_config, size, packets):
```

Comment:
We can get rid of `raw_traffic` in test and file names.

Reply:
done